spec/features/transactions_filter_spec.rb

```diff
@@ -27,7 +27,7 @@
 
     before do
       visit root_path
-      click_on 'Filters'
+      find(:css, 'a#toggle_filter').trigger('click')
```

Comment:
@shir Capybara::Webkit::ClickFailed: Failed to click element /html/body/div/div[2]/div[1]/h2/a[@id='toggle_filter'] because of overlapping element /html/body/div/div[2]/div[1]/h2